stock_operating_unit/__openerp__.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# © 2016 Eficent Business and IT Consulting Services S.L.
+# © 2016 Serpent Consulting Services Pvt. Ltd.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+{
+    "name": "Stock with Operating Units",
+    "summary": "An operating unit (OU) is an organizational entity part of a\
+        company",
```

Comment:
The following form would be better:

```
    "summary": "An operating unit (OU) is an organizational entity part of a "
               "company",
```

Because ithe current form includes a lot of spaces before "company".